.github/pull_request_assignment.yml

```diff
@@ -103,108 +85,24 @@
     paths:
       - "specification/network/**"
```

Comment:
>"specification/network/**" [](http://example.com/codeflow?start=8&length=26)

change to "specification/network/resource-manager/**". Similarly other places

Reply:
network, sql, and standbypool all of them only contain control plane swaggers.